### ISSUE TITLE
«Call to a member function getEmail() on null in vendor/mage2pro/core/Paypal/Plugin/Model/Api/Nvp.php:43»

### ISSUE THREAD

**Issue by dmitrii-fediuk** [royalwholesalecandy/core#41]

**Comment by dmitrii-fediuk** [comment]
```
#0 vendor/mage2pro/core/Paypal/Plugin/Model/Api/Nvp.php(31): Df\Paypal\Plugin\Model\Api\Nvp::eligible(NULL)
#1 vendor/magento/framework/Interception/Interceptor.php(121): Df\Paypal\Plugin\Model\Api\Nvp->beforeCallSetExpressCheckout(Object(Magento\Paypal\Model\Api\Nvp\Interceptor))
#2 vendor/magento/framework/Interception/Interceptor.php(153): Magento\Paypal\Model\Api\Nvp\Interceptor->Magento\Framework\Interception\{closure}()
#3 generated/code/Magento/Paypal/Model/Api/Nvp/Interceptor.php(65): Magento\Paypal\Model\Api\Nvp\Interceptor->___callPlugins('callSetExpressC...', Array, Array)
#4 vendor/magento/module-paypal/Model/Express/Checkout.php(567): Magento\Paypal\Model\Api\Nvp\Interceptor->callSetExpressCheckout()
```

**Comment by dmitrii-fediuk** [comment]
Fixed in [5.5.8](https://github.com/mage2pro/core/releases/tag/5.5.8).
